FandomParser/FandomParser/WikiBuildingInfoProvider.cs

```diff
@@ -149,13 +149,13 @@ private static ConstructionInfo ParseContainingInfoIcon(string constructionCost)
         {
             ConstructionInfo result = null;
 
-            if (!double.TryParse(constructionCost.Split(new[] { "{{" }, StringSplitOptions.RemoveEmptyEntries)[0], out double value))
-            {
+            var splittedInfo = constructionCost.Split(new[] { "{{" }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Maybe a constant should be defined for `{{`?

Reply:
Shouldn't make mauch of a difference because the compiler will make it a contant anyway, but I changed it.